### PR TITLE
Chain requests when running the collection

### PR DESCRIPTION
**Pull request by ben741**
Use Tests to set the order of requests when using the "Run collection"
feature, including polling status requests until they finish.

See https://learning.postman.com/docs/running-collections/building-workflows/

### CODE REVIEW

File: Zuva DocAI.postman_collection.json [hunk 1]
```diff
@@ -25,6 +24,9 @@
 									"pm.test(\"The response contains a file_id.\", function () {",
 									"    pm.expect(jsonData).to.have.property('file_id');",
 									"    pm.environment.set(\"file_id\", jsonData[\"file_id\"]);",
+									"",
+									"    // Chaining for when we run the whole collection",
+									"    postman.setNextRequest(\"Create OCR Request\")",
```
Comment (matt-cote) [inline Zuva DocAI.postman_collection.json:29]: Just making a mental note that they are chained by name so if we ever rename any of the requests we'll have to remember to update these as well.

File: Zuva DocAI.postman_collection.json [hunk 2]
```diff
@@ -2303,7 +2394,7 @@
 								"exec": [
 									"jsonData = pm.response.json()",
 									"",
-									"pm.test(\"The document classification request status was successful.\", function () {",
+									"pm.test(\"The document classification request status was  successfully retrieved.\", function () {",
```
Comment (matt-cote) [inline Zuva DocAI.postman_collection.json]: Nitpick but it looks like we've introduced a double space here.

File: Zuva DocAI.postman_collection.json [hunk 3]
```diff
@@ -2527,14 +2641,31 @@
 								"exec": [
 									"jsonData = pm.response.json()",
 									"",
-									"pm.test(\"The language classification request status was successful.\", function () {",
+									"pm.test(\"The language classification request status was  successfully retrieved.\", function () {",
```
Comment (matt-cote) [inline Zuva DocAI.postman_collection.json]: Same here, double space.

File: Zuva DocAI.postman_collection.json (hunk 1 above)
Comment (ben741) [reply Zuva DocAI.postman_collection.json:29]: Yeah, it's not great. Unfortunately the only other way is by request ID, but the IDs are lost when the collection is exported. See https://github.com/postmanlabs/postman-app-support/issues/5908#issuecomment-1062614465

We could switch to those request IDs if we drop the Github version of this, but human-readable names are also preferable to random GUIDs in terms of understanding what's going on anyways.

**Review by matt-cote — APPROVED** [review]

**Pull request merged**